server/container_create_linux.go

```diff
@@ -867,14 +769,18 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrIface.Contai
 
 	sort.Sort(orderedMounts(mounts))
 
+	rspecMount := rspec.Mount{
+		Destination: "",
+		Type:        "bind",
+		Source:      "",
+		Options:     nil,
+	}
 	for _, m := range mounts {
-		mnt = rspec.Mount{
-			Type:        "bind",
-			Source:      m.Source,
-			Destination: m.Destination,
-			Options:     append(m.Options, "bind"),
-		}
-		specgen.AddMount(mnt)
+		options := append(m.Options, "bind")
+		rspecMount.Destination = m.Destination
+		rspecMount.Source = m.Source
+		rspecMount.Options = options
```

Comment:
nit: `rspecMount.Options = append(m.Options, "bind")`

Reply:
i.e. there's no need to use a temp var `options`

Reply:
fixed!